src/labels/CustomLabels.labels

```diff
@@ -1623,7 +1623,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Label indicating user can hide the credit card and not use Elevate</shortDescription>
-        <value>Do Not Use Elevate</value>
+        <value>Don&apos;t enter payment information</value>
```

Comment:
@sam-knox , @mpusto : 
@psadie-sfdo  indicated the same content will be also applicable for GE entry (to be released in `feature/230` I believe). I think we can leave the labels for now, but we might want to merge later on? Thoughts?
CC @vmartinov-sforg , @melissabarber

Reply:
Agree that the labels are fine for the time being. From my perspective, we'd ideally rename these with the `ps` prefix and retire the other.